src/App.js

```diff
@@ -3,18 +3,32 @@ import "./App.css";
 import Announcements from "./components/Announcements";
 import Inventory from "./components/Inventory";
 import Orders from "./components/Orders";
+import { Route, Switch, Redirect } from "react-router-dom";
+import Login from "./components/Login";
 import NavigationBar from "./components/NavigationBar";
-import { Route, Switch } from "react-router-dom";
 
 function App() {
   return (
     <div className="App">
-      <NavigationBar />
-      <Switch>
-        <Route component={Inventory} path="/inventory" />
-        <Route component={Orders} path="/orders" />
-        <Route component={Announcements} path="/" />
-      </Switch>
+      <div>
+        <Switch>
+          <Route path="/login" component={Login} />
+          <Redirect from exact="/" to="/login" />
```

Comment:
why not just do <Route path="/" component={Login} /> at the bottom of the switch statement.